src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -0,0 +1,206 @@
+        bool ActiveChangeCallbacks { get; }
+        bool HasChanged { get; }
+        System.IDisposable RegisterChangeCallback(System.Action<object> callback, object state);
+    }
+    [System.Diagnostics.DebuggerDisplayAttribute("Value = {_value}")]
+    [System.ObsoleteAttribute("This type is obsolete and will be removed in a future version.")]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct InplaceStringBuilder
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public InplaceStringBuilder(int capacity) { throw null; }
+        public int Capacity { get { throw null; } set { } }
+        public void Append(Microsoft.Extensions.Primitives.StringSegment segment) { }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public void Append(char c) { }
+        public void Append(string value) { }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public void Append(string value, int offset, int count) { }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct StringSegment : System.IEquatable<Microsoft.Extensions.Primitives.StringSegment>, System.IEquatable<string>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public static readonly Microsoft.Extensions.Primitives.StringSegment Empty;
+        public StringSegment(string buffer) { throw null; }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public StringSegment(string buffer, int offset, int length) { throw null; }
+        public string Buffer { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public bool HasValue { get { throw null; } }
+        public char this[int index] { get { throw null; } }
+        public int Length { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public int Offset { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public string Value { get { throw null; } }
+        public System.ReadOnlyMemory<char> AsMemory() { throw null; }
+        public System.ReadOnlySpan<char> AsSpan() { throw null; }
+        public static int Compare(Microsoft.Extensions.Primitives.StringSegment a, Microsoft.Extensions.Primitives.StringSegment b, System.StringComparison comparisonType) { throw null; }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public bool EndsWith(string text, System.StringComparison comparisonType) { throw null; }
```

Comment:
I don't thin we need this either. It seems like you didn't pass this list to GenAPI when generating the source. https://github.com/dotnet/runtime/blob/master/eng/DefaultGenApiDocIds.txt